packages/components/src/Modal/Modal.tsx

```diff
@@ -69,12 +71,13 @@ export function Modal({
               stiffness: 300,
             }}
           >
-            <Header
-              title={title}
-              dismissible={dismissible}
-              onRequestClose={onRequestClose}
-            />
-
+            {showTitle && (
```

Comment:
Just thinking it might be nice to do

```suggestion
            {title != undefined && (
```

here.

Reply:
Let's do it!